### PR TITLE
GSoC'21 Mid-Term Progress: Aitik Gupta

### PR DESCRIPTION
**Pull request by aitikgupta**
We're almost halfway through!

image

### CODE REVIEW

**Comment by davidevaleriani** [comment]
@story645 do you want to check this before me merging?

File: content/posts/GSoC_2021_MidTerm/index.md
Comment (story645) [inline content/posts/GSoC_2021_MidTerm/index.md]: Not sure this metaphor is helping explain your point 'cause you don't go deep enough to explain why gdocs being all things is bad (and not sure you should cause it'll distract)

File: content/posts/GSoC_2021_MidTerm/index.md
Comment (story645) [inline content/posts/GSoC_2021_MidTerm/index.md]: Maybe just relink the guide?

**Review by story645 — COMMENTED** [review]
Great post, like the balance of detail and whole picture. Couple of nits.

File: content/posts/GSoC_2021_MidTerm/index.md
Comment (aitikgupta) [reply content/posts/GSoC_2021_MidTerm/index.md]: Ah, okay. I can remove this whole analogy itself 👍🏼

File: content/posts/GSoC_2021_MidTerm/index.md
Comment (aitikgupta) [reply content/posts/GSoC_2021_MidTerm/index.md]: 👍🏼

**Review by story645 — APPROVED** [review]

**Comment by aitikgupta** [comment]
Mind if I bump this up?
I need to add the deployed link to the spreadsheet 😄

**Comment by davidevaleriani** [comment]
Sure, let me know when ready to publish

**Comment by aitikgupta** [comment]
I think it's ready to publish, since @story645 has approved after I worked on her review.

**Comment by story645** [comment]
Yeah sorry only reason I didn't merge is cause every time I try, something usually goes wrong

**Pull request merged**
